src/platform/plugins/shared/esql/public/plugin.ts

```diff
@@ -18,11 +18,11 @@ import type { UsageCollectionStart } from '@kbn/usage-collection-plugin/public';
 import { Storage } from '@kbn/kibana-utils-plugin/public';
 import {
   updateESQLQueryTrigger,
-  UpdateESQLQueryAction,
   UPDATE_ESQL_QUERY_TRIGGER,
   esqlControlTrigger,
-  CreateESQLControlAction,
   ESQL_CONTROL_TRIGGER,
+  ACTION_UPDATE_ESQL_QUERY,
+  ACTION_CREATE_ESQL_CONTROL,
 } from './triggers';
```

Comment:
Don't import entire index file in plugin. This loads all resources into page load bundle. Instead, just directly import the items needed from the there individual files.

Reply:
I did but I dont see any changes in the metrics